tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/IOperationContext.cs

```diff
@@ -3,21 +3,19 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using Datadog.Trace.DuckTyping;
+
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
 {
     /// <summary>
     /// HotChocolate.Execution.Processing.IOperationContext interface for ducktyping
     /// </summary>
-    internal interface IOperationContext
+    [DuckCopy]
+    internal struct IOperationContext
     {
         ///// <summary>
         ///// Gets the context operation
         ///// </summary>
-        IPreparedOperation Operation { get; }
-
-        ///// <summary>
-        ///// Gets the operation Result
-        ///// </summary>
-        IResultHelper Result { get; }
+        public IPreparedOperation Operation { get; }
```

Comment:
This is going to fail because when using the `[DuckCopy]` struct you have to change the get properties to a public fields in order to make it work.

Also as a nit, I prefer change the naming and remove the `I` prefix due is not longer an interface. Consider using something like `OperationContextStruct` or just check what other integrations are doing.

Reply:
You are right with the naming. It was pending but pushed accidentally